source/pug/partials/events.pug

```diff
@@ -4,7 +4,9 @@
     h4 Launch events
 
 .container
-  include:markdown ../../markdown/events.md
+  .row.justify-content-center
+    .col-12.col-xl-10
```

Comment:
Looking at the rest of the page, this PR would make it out of place, so I think I'm just going to close it. But it's here for reference in case broader changes are desired.